Configuration Files/4MotorTest.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version='1.0' encoding='UTF-8' standalone='yes' ?>
+<Robot type="FirstInspires-FTC">
+    <LynxUsbDevice name="Expansion Hub Portal 1" serialNumber="DQ16OY71" parentModuleAddress="2">
+        <LynxModule name="Expansion Hub 2" port="2">
+            <NeveRest40Gearmotor name="rightfront" port="0" />
+            <NeveRest40Gearmotor name="leftfront" port="1" />
+            <NeveRest40Gearmotor name="rightrear" port="2" />
+            <NeveRest40Gearmotor name="leftrear" port="3" />
+            <Servo name="rightArm" port="0" />
+            <LynxEmbeddedIMU name="imu" port="0" bus="0" />
+            <LynxColorSensor name="colorDistanceRight" port="0" bus="1" />
+        </LynxModule>
+    </LynxUsbDevice>
+</Robot>
```

Comment:
#AnswerPlease Interesting! Can you provide some background on configuration files as xml?

Reply:
Unfortunately, I'm not sure how the configuration files are structured. I simply added the file to the program so it would be readily available in case one of the phones lost their configuration files, or if a new configuration is made and needs to be transfered to the other set of phones.

Reply:
Ah ok, that makes sense, good idea.